src/Mjolnir.ts

```diff
@@ -413,7 +406,7 @@ export class Mjolnir {
         // We filter out all policy rooms so that we only protect ones that are
         // explicitly protected, so that we don't try to protect lists that we are just watching.
         const filterOutManagementAndPolicyRooms = (roomId: string) => {
-            const policyListIds = this.policyLists.map(list => list.roomId);
+            const policyListIds = this.policyListManager.lists.map(list => list.roomId);
             return roomId !== this.managementRoomId && !policyListIds.includes(roomId);
         };
```

Comment:
It is a smell that this filter has been ignored and not included as part of the `PolicyListManager`.

Reply:
Ok, that's because it's to do with protected rooms, not the policy list manager... maybe it could be moved there though?